uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -169,8 +160,34 @@ define([
          *
          * @return {Promise} resolved with {Boolean}
          */
-        isSentBack: function() {
+        isSentBack: function () {
           return checkLeaveStatus.call(this, 'more_information_requested');
+        },
+
+        /**
+         * Check the role of a given contact in relationship to the leave request.
+         *
+         * @param {Object} contact - contact object
+         *
+         * @return {Promise} resolves with an {String} - owner/manager/none
+         */
+        roleOf: function (contact) {
+          var deferred = $q.defer();
+
+          if (this.contact_id === contact.id) {
```

Comment:
Maybe it's better `==`, we would avoid problems in case one of the ids is a string